src/data/restaurants.hql

```diff
@@ -4,10 +4,8 @@ SET hive.groupby.orderby.position.alias=true;
 
 ADD JAR /home/hadoop/Yelp/json-serde-1.3.7-jar.jar;
 
-DROP TABLE IF EXISTS restaurants;
```

Comment:
Just below it I changed CREATE TABLE to CREATE TABLE IF NOT EXISTS
That way the table only has to be created once.